docs/UserGuide.md

```diff
@@ -88,32 +89,44 @@ What can I do for you?
       Updates the first expenditure on your expenditure list to reflect a $4.50 expenditure of the description 'Chicken
       Rice' that was paid in cash on 12 March 2022.
     * **`delete`**`/e 1` : Deletes the first expenditure in your expenditure list.
-    * **`bye`** : Exits the app.
+    * **`bye`** : Exits the application.
 
 6. Refer to the [Features](#features) for more details of each command.
 
 <br/>
 
 # Features
 
-The following are features of the `MindMyMoney` application. Please ensure that the format of commands given is
-accurate.  
-Words in `[SQUARE_BRACKETS]` are compulsory parameters. Words in `{CURLY_BRACKETS}` are optional parameters. <br/>
-Words starting with a `/` are flags.
+The following are features of the `MindMyMoney` application.
+
+## Introduction to Commands
+
+A `MindMyMoney` command typically contains 4 types of terms.
+
+- The first term in the command is known as the instruction.
+- Terms in `[SQUARE_BRACKETS]` are compulsory parameters.
+- Terms in `{CURLY_BRACKETS}` are optional parameters.
+- Terms starting with a `/` are flags.
+
+For Example:
+
+- `list /e {DATE}`
+    - `list` is an instruction, `/e` is a flag and `{DATE}` is an optional parameter.
+- `calculate /epm [DATE]`
+    - `calculate` is an instruction, `/epm` is a flag and `[DATE]` is a compulsory parameter.
 
 > **💡 Note:**
 >- Parameters and flags are space-separated. For example: `list/e` is not a valid command while `list /e` is valid.
->- 
 
 > **⚠️Warning⚠️**
->- All parameters are compulsory unless stated otherwise.
->- Input the parameters in the order shown, or the application will not be able to read your
+>- Input the parameters in the order shown in the guide, else the application will not be able to read your
    > input.
->- When entering your input, use only ascii characters. Any other input may not properly display in the command line.
+>- Only use ascii characters when entering commands into the application. Otherwise, your input may not be properly displayed in the command line.
```

Comment:
nitpick but caps. Also, not sure if we should explain what ASCII is
```suggestion
>- Only use ASCII characters when entering commands into the application. Otherwise, your input may not be properly displayed in the command line.
```